db/schema.rb

```diff
@@ -29,6 +29,18 @@
     t.index ["user_id"], name: "index_batch_contexts_on_user_id"
   end
 
+  create_table "bundle_contexts", force: :cascade do |t|
+    t.string "project_name", null: false
+    t.integer "content_structure", null: false
+    t.string "bundle_dir", null: false
+    t.boolean "staging_style_symlink", default: false, null: false
+    t.integer "content_metadata_creation", null: false
+    t.integer "user_id", null: false
+    t.datetime "created_at", precision: nil, null: false
+    t.datetime "updated_at", precision: nil, null: false
+    t.index ["user_id"], name: "index_bundle_contexts_on_user_id"
+  end
+
```

Comment:
where is this table creation coming from?

Reply:
oops - mistake from a previous bad migration when developing, fix the problem with my test database and this went away